ml_tools/eolearn/tests/test_train_split.py

```diff
@@ -36,58 +36,65 @@ def test_bad_args(bad_arg: Any, bad_kwargs: Any) -> None:
         TrainTestSplitTask(INPUT_FEATURE, OUTPUT_FEATURE, bad_arg, **bad_kwargs)
 
 
-def test_train_split() -> None:
-    shape = (1000, 1000, 3)
-    data = np.random.randint(10, size=shape, dtype=int)
+@pytest.fixture(name="eopatch1")
+def eopatch1_fixture(seed: Optional[int] = None) -> EOPatch:
+    eopatch = EOPatch()
+    rng = np.random.default_rng(seed)
```

Comment:
seed is never given as input, consider using a SEED constant instead (using None applies a random random seed)

Reply:
done

Reply:
since you're never passing in anything for the `seed` parameter you might as well remove it from both fixtures and use SEED direclty

Reply:
done